bundles/org.eclipse.passage.lbc.equinox/src/org/eclipse/passage/lbc/internal/equinox/ServerRunnerImpl.java

```diff
@@ -65,8 +66,21 @@ public void unbindLogger(LoggerFactory factory) {
 	public void bindServerHandler(BackendLauncher serverHandler, Map<String, Object> context) {
 		if (serverHandler != null) {
 			logger.info(NLS.bind(EquinoxMessages.ServerRunnerImpl_i_launcher_bind, serverHandler));
-			serverHandler.launch(context);
-			backendLaunchers.add(serverHandler);
+			LicensingResult result = serverHandler.launch(context);
+			switch (result.getSeverity()) {
+			case LicensingResult.ERROR:
+				logger.error(result.getMessage());
+				logger.error(
+						NLS.bind(EquinoxMessages.ServerRunnerImpl_error_launching, serverHandler.getClass().getName()));
+				break;
+			case LicensingResult.OK:
+				logger.info(result.getMessage());
+				backendLaunchers.add(serverHandler);
+				break;
+			default:
+				break;

```

Comment:
This is a problem to be resolved here: over severity values of results should be also handled and default should not be empty